do_freebsd.sh

```diff
@@ -32,6 +32,7 @@ rm -rf build && ./do_cmake.sh "$*" \
 	-D WITH_CEPHFS=OFF \
 	-D WITH_EMBEDDED=OFF \
 	-D WITH_MGR=YES \
+	-D WITH_SPDK=OFF \
```

Comment:
nit, please squash this commit into fa1d09e950712bc2f9c515e7236459e08892bfa6

Reply:
agree

Reply:
addressed.